addons/zylann.debug_draw/debug_draw.gd

```diff
@@ -373,7 +372,7 @@ func _on_CanvasItem_draw():
 		pos.y += line_height
 
 
-static func _create_wirecube_mesh(color := Color.WHITE) -> ArrayMesh:
+func _create_wirecube_mesh(color := Color.WHITE) -> ArrayMesh:
```

Comment:
Which warning is this fixing? Theres shouldn't be any issue with this function being static. Making it non-static removes the explicit intent of this function being standalone.
If that warning is Godot complaining that the static method is called without the `Class.method` syntax, then it's one of the dumbest warnings Godot has, especially for this class which can't even refer to itself without simply omitting the class name. Or maybe it's a bug.

Reply:
The warning is the following, which happens on single place where `_create_wirecube_mesh` is called:

```
W 0:00:00:0873   The function "_create_wirecube_mesh()" is a static function but was called from an instance. Instead, it should be directly called from the type: "CanvasLayer._create_wirecube_mesh()".
  <GDScript Error>STATIC_CALLED_ON_INSTANCE
  <GDScript Source>debug_draw.gd:195
```

Reply:
It should not be necessary to add more stuff to this, I believe that warning is bogus